### ISSUE TITLE
Temporary file extension

### ISSUE THREAD

**Issue by massiliattak** [FallingSnow/h265ize#77]
Hello,
Could you add a feature to h265ize that makes the output file to have a temp extension (e.g.: ".h265tmp"), and rename it to mkv at the end of the encoding process?
This is in order to prevent scripts/applications to post-process/move unfinished encoded files.

Thank you.

**Comment by FallingSnow** [comment]
Yes, h265ize used to do this. When I did the rewrite it ended up getting removed.
